test/jdk/sun/tools/jcmd/TestJcmdPIDSubstitution.java

```diff
@@ -47,7 +47,6 @@ public class TestJcmdPIDSubstitution {
     public static void main(String[] args) throws Exception {
         verifyOutputFilenames("Thread.dump_to_file", FILENAME);
         verifyOutputFilenames("GC.heap_dump", FILENAME);
-        verifyOutputFilenames("Compiler.perfmap", FILENAME);
```

Comment:
FWIW this could have been guarded by a platform check.

Reply:
System.dump_map also is only supported on linux.